lib/src/main/java/com/github/featuredetectlib/traditional/NativeKeypointWrapper.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.featuredetectlib.traditional
+
+import com.github.featuredetectlib.Keypoint
+
+/**
+ * Wrapper for a native keypoint.
+ */
+internal data class NativeKeypointWrapper(private val keypoint: StrippedKeypoint) : Keypoint {
+    override val x: Float
+        get() = keypoint.x
+    override val y: Float
+        get() = keypoint.y
+    override val strength: Float
+        get() = keypoint.strength
+    override val size: Float
+        get() = keypoint.size
+    override val angle: Float?
+        get() = keypoint.angle.takeIf { it != -1f } // angle == -1 if not applicable
```

Comment:
Maybe we can make `-1f` a constant (or something like that)? Or there will be troubles with bridge code generation and typecasts?

Reply:
Good suggestion, everything'll be alright, will do